compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion/Encoding.hs

```diff
@@ -0,0 +1,106 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- | Encoding/decoding of metadata (i.e. non-semantically-relevant bindings) in LF,
+-- such as functional dependencies and typeclass instance overlap modes. These are
+-- added in during LF conversion, and then decoded during data-dependencies to
+-- improve the reconstructed module interface.
+module DA.Daml.LFConversion.Encoding
```

Comment:
`Encoding` is so generic. Let's be a bit more specific, maybe
```suggestion
module DA.Daml.LFConversion.MetadataEncoding
```

Reply:
Ah, I like it. I had "Metadata" at first but it was also too generic. This is perfect. (-: